### PR TITLE
Testing : logrotate permissions + v0.12.2 + package v2

### PR DESCRIPTION
**Pull request by lapineige**
Fix https://github.com/YunoHost-Apps/shaarli_ynh/issues/83

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
I can't resolve conflicts myself.

**Comment by lapineige** [comment]
ping @yalh76 @ericgaspar for review :)

File: doc/ADMIN.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 During the installation app will prompt you to install itself as public or private, if you choose public then anyone (not just YunoHost users) would be able to access your Shaarli instance.
```
Comment (alexAubin) [inline doc/ADMIN.md:3]: This bit ain't really useful, YunoHost's explanation from the core should already be okay-ish (or to be improved for all apps, not just this one in particular ... but cf also the already existing `help` key in `manifest.toml`) AND .. ADMIN.md is meant to be displayed after install and meant to contain "administration documentation" so not really relevant to have this info which is relevant only prior install

```suggestion
```

File: doc/ADMIN.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
...
 
-After the installation is complete, go to the domain where the Shaarli is installed and create your account. After the creation of the account the registration will be locked.
\ No newline at end of file
+After the installation is complete, go to https://__DOMAIN____PATH__ and create your account. After the creation of the account the registration will be locked.
```
Comment (alexAubin) [inline doc/ADMIN.md:5]: We should probably move this bit to `POST_INSTALL.md` rather than here

File: scripts/upgrade [hunk 3]
```diff
@@ -79,34 +27,26 @@ then
...
 
 	# Backup the config file in the temp dir
-	cp -a "$final_path/data" "$tmpdir/data"
-	cp -a "$final_path/tpl" "$tmpdir/tpl"
+	cp -a "$install_dir/data" "$tmpdir/data"
+	cp -a "$install_dir/tpl" "$tmpdir/tpl"
 
 	# Remove the app directory securely
-	ynh_secure_remove "$final_path"
+#REMOVEME? 	ynh_secure_remove "$install_dir"
```
Comment (alexAubin) [inline scripts/upgrade:34]: Hmmm, should it be removed or ... should it be removed ? :sweat_smile:

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
Review required :)

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
!testme

**Comment by lapineige** [comment]
Please review and merge this PR 🙂

**Review by alexAubin — APPROVED** [review]

**Comment by alexAubin** [comment]
LGTM, YOLO !

**Comment by lapineige** [comment]
YOLOhost :grin:

**Pull request merged**